stl/inc/xloctime

```diff
@@ -666,6 +666,57 @@ _NODISCARD constexpr bool _Is_valid_strftime_specifier(const char _Specifier) {
     return false;
 }
 
+_NODISCARD constexpr bool _Is_valid_strftime_tm_data(const tm* _Pt) {

```

Comment:
`_Pt` could be top-level `const`. We're not totally disciplined about this but we try to use `const` as much as possible in new code.

Reply:
I'm not sure I understand the comment. It is already `const`, so I'm not sure how to interpret "top-level" `const`. Perhaps you mean to also make the pointer const, not just the data, like `const tm* const _Pt`?

Reply:
Yes, that's what top-level means - `const` that applies to the pointer itself, instead of what it points to.